dependencies.json

```diff
@@ -150,11 +150,11 @@
         }
       },
       "fontAwesome": {
-        "version": "5.9.0",
+        "version": "6.4.2",
```

Comment:
(Confirming) Have you followed [the wiki page](https://github.com/oppia/oppia/wiki/Third-party-libraries#upgrade-frontend-libraries-not-installed-from-npm) to updagrade this lib?

Also, can you confirm this will not affect anny other part of the codebase?

Reply:
Hi,I have reverted the upgrade,so now it is the previous one only(ie. 5.9.0)
Please check the commit named:**Changes after review.**